volestipy/volestipy/volestipy.pyx

```diff
@@ -115,3 +126,86 @@ cdef class HPolytope:
    @property
    def dimensions(self):
       return self._A.shape[1]
+
+
+
+
+# build the low_dim_polytope_cpp class
+cdef class low_dim_HPolytope:
+
+   cdef lowDimHPolytopeCPP low_dim_polytope_cpp
+   cdef double[:,::1] _A
+   cdef double[::1] _b
+   cdef double [:,::1] _Aeq
+   cdef double[::1] _beq
+
+# set the specs of the class
+   def __cinit__(self, double[:,::1] A, double[::1] b, double[:,::1] Aeq, double[::1] beq):
+      self._A = A
+      self._b = b
+      self._Aeq = Aeq
+      self._beq = beq
+      n_rows_of_A, n_cols_of_A = A.shape[0], A.shape[1] 
+      n_row_of_Aeq, n_cols_of_Aeq = Aeq.shape[0], Aeq.shape[1] 
```

Comment:
Please check if:
1) n_rows_of_A == b.shape[0]
2) n_row_of_Aeq == beq.shape[0]
3) n_cols_of_A == n_cols_of_Aeq

Reply:
Three if statements were added to check for these equalities